src/Tasks/GenerateResource.cs

```diff
@@ -2988,7 +2994,7 @@ private void ReadResources(String filename, bool shouldUseSourcePath, String out
 #if FEATURE_ASSEMBLY_LOADFROM
                 ReadAssemblyResources(filename, outFileOrDir);
 #else
-                _logger.LogError("Reading resources from Assembly not supported on .NET Core MSBuild");
+                throw new InputFormatNotSupportedException("Reading resources from Assembly not supported on .NET Core MSBuild");
```

Comment:
Why is it advantageous to throw instead of just logging an error? Are you trying to reduce duplicate output?

Reply:
Logging an error here is nonfatal, so the experience with going through this codepath was:

1. This error.
2. Some other odd output when trying to move on.
3. Another fatal error that didn't directly indicate the problem.

Since there were already exception-based error paths in `ProcessFile` I opted to add another rather than change everything to error returns.